deploy/src/main/deploy/helm/values.yaml

```diff
@@ -109,6 +126,9 @@ adapters:
   #deviceConnectionSpec:
   #deviceRegistrationSpec:
   #tenantSpec:
+    svc:
```

Comment:
I believe this belongs to the AMQP adapter config, right? If so, then it needs to go *before* `#amqpMessagingNetworkSpec` ...